extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/ContractDefinitionRequestDto.java

```diff
@@ -0,0 +1,36 @@
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import de.sovity.edc.ext.wrapper.api.common.model.CriterionDto;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class ContractDefinitionRequestDto {
+
+    /**
+     * Default validity is set to one year.
+     */
+    private static final long DEFAULT_VALIDITY = TimeUnit.DAYS.toSeconds(365);
+
+    private String id;
+    @NotNull(message = "accessPolicyId cannot be null")
+    private String accessPolicyId;
+    @NotNull(message = "contractPolicyId cannot be null")
+    private String contractPolicyId;
+    @Valid
+    @NotNull(message = "criteria cannot be null")
+    private List<CriterionDto> assetsSelector;
+
```

Comment:
These annotations don't work. Need `@Schema` with documentation and required

Reply:
https://github.com/sovity/edc-extensions/issues/471